shared/components/activityCard.js

```diff
@@ -0,0 +1,283 @@
+      </View>
+    );
+  }
+
+  renderEventOption() {
+    return (
+      <View style = {{flex: 111, backgroundColor: '#FCFCFC', flexDirection: 'row'}}>
+        <View style = {{flex: 16}}/>
+        <View style = {{flex: 344}}>
+          <View style = {{flex: 16}}/>
+          <View style = {{flex: 24, flexDirection: 'row'}}>
+            <View style = {{flex: 24}}>
+              <Image source = {IMG_BUTTON_EDIT} style = {{width: 24, height: 24}}/>
+            </View>
+            <View style = {{flex: 8}}/>
+            <View style = {{flex: 272}}>
+              <Text style = {styles.option}>Edit Suggestion</Text>
+            </View>
+            <View style = {{flex: 24}}>
+              <Image source = {IMG_BUTTON_MORE} style = {{width: 24, height: 24}}/>
+            </View>
+            <View style = {{flex: 16}}/>
+          </View>
+          <View style = {{flex: 16}}/>
+          <View style = {{flex: 22, flexDirection: 'row'}}>
+            <View style = {{flex: 24}}>
+              <Image source = {IMG_BUTTON_RED_FLAG} style = {{width: 24, height: 24}}/>
+            </View>
+            <View style = {{flex: 8}}/>
+            <View style = {{flex: 312}}>
+              <View style = {{flex: 1 }}>
+                <Text style = {styles.option}>Event Reported</Text>
+              </View>
+            </View>
+          </View>
+          <View style = {{flex: 8}}/>
+          <View style = {{flex: 14}}/>
+          <View style = {{flex: 16}}/>
+        </View>
+      </View>
+    );
+  }
+
+  transformTodate(rowData) {
+    let startTime = new Date(rowData.startTime);
+    let endTime = '';
+    if (rowData.endTime) {
+      endTime = new Date(rowData.endTime);
+    }
+    let date = '';
+    let monthNames = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'June',
+      'July', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'
+    ];
+    let meridiem = 'am';
+    let hour = startTime.getHours();
+    if (hour === 0) {
+      hour = 12;
+    } else if (hour === 12) {
+      meridiem = 'pm';
+    } else if (hour > 12) {
+      hour %= 12;
+      meridiem = 'pm';
+    }
+    if (hour < 10) {
+      hour = '0' + hour;
+    }
+    let minute = startTime.getMinutes();
+    if (minute === 0) {
+      minute = '00';
+    } else if (minute < 10) {
+      minute = '0' + minute;
+    }
+    date += monthNames[startTime.getMonth()] + '. ' + startTime.getDate() + ', ';
+    date += hour + ':' + minute + meridiem + ' - ';
+    if (endTime) {
+      meridiem = 'am';
+      hour = endTime.getHours();
+      if (hour === 0) {
+        hour = 12;
+      } else if (hour === 12) {
+        meridiem = 'pm';
+      } else if (hour > 12) {
+        hour %= 12;
+        meridiem = 'pm';
+      }
+      if (hour < 10) {
+        hour = '0' + hour;
+      }
+      minute = endTime.getMinutes();
+      if (minute === 0) {
+        minute = '00';
+      } else if (minute < 10) {
+        minute = '0' + minute;
+      }
+      date += hour + ':' + minute + meridiem;
```

Comment:
This code are duplicated. 
I think you can make it as a module and change the duplicated code to function call